# Module 71.1: Arbitrage, Replication, and Carrying Costs

<hr>

### LOS 71.a: Explain how the concepts of arbitrage and replication are used in pricing derivatives.

In contrast to valuing risky assets as the (risk-adjusted) present value of expected future cash flows, the valuation of derivative securities is based on a **no-arbitrage condition**. *Arbitrage* refers to a transaction in which an investor purchases one asset or portfolio of assets at one price and simultaneously sells an asset or portfolio of assets that has the same future payoffs, regardless of future events, at a higher price, realizing a risk-free gain on the transaction.

While arbitrage opportunities may be rare, the reasoning is that when they do exist, they will be exploited rapidly. Therefore, we can use a no-arbitrage condition to determine the current value of a derivative, based on the known value of a portfolio of assets that has the same future payoffs as the derivative, regardless of future events. Because there are transaction costs of exploiting an arbitrage opportunity, small differences in price may persist when the arbitrage gain is less than the transaction costs of exploiting it.

We can illustrate no-arbitrage pricing with a 1-year forward contract, with a forward price of $F_0(1)$, on an Acme share that pays no dividends and is trading at a current price, $S_0$, of \$30.


Consider two strategies to own an Acme share at $t=1$:

* *Portfolio 1*: Buy a pure discount bond with a yield of 5\% that pays $F_0(1)$ at $t = 1$. The current cost of the bond is $F_0(1) \div 1.05$. Additionally, enter a forward contract on one Acme share at $F_0(1)$ as the buyer. The forward has a zero cost, so the cost of Portfolio 1 is $F_0(1) \div 1.05$.
\
\
At $t = 1$ the bond pays $F_0(1)$, which will buy an Acme share at the forward price, so that the payoff on Portfolio 1 is the value of one share at $t = 1$, $S_1$.

\
* *Portfolio 2*: Buy a share of Acme at $S_0 = 30$ and hold it for one year. Cost at $t = 0$ is $\$30$.
\
\
At $t = 1$ the value of the Acme share is $S_1$, and this is the payoff for 


The no-arbitrage condition (law of one price) requires that two portfolios with the same payoff in the future for any future value of Acme have the same cost today. Because our two portfolios have a payoff of $S_1$, they must have the same cost at $t = 0$ to prevent arbitrage. That is, $F_0(1) \div 1.05 = \$30$, so we can solve for the no-arbitrage forward price as $F_0(1) = 30(1.05) = 31.50$.

To better understand the no-arbitrage condition, we will consider two situations in which the forward price is not at its no-arbitrage value: $F_0(1) > 31.50$ and $F_0(1) < 31.50$.


* If the forward contract price is $32 (F0_(1) > 31.50)$, the profitable arbitrage is to sell the forward (because the forward price is "too high") and buy a share of stock. At $t = 1$, deliver the share under the forward contract and receive $32$, for a return of $32/30 – 1 = 6.67\%$, which is higher than the risk-free rate.
\
\
We can also view this transaction as borrowing $30$ at the risk-free rate $(5\%)$ to buy the Acme share at $t = 0$ and at $t = 1$ paying $31.50$ to settle the loan. The share delivered under the forward has a contract price of $32$, so the arbitrageur has an arbitrage profit of $32 – 31.50 = 0.50$ with no risk and no initial cost.
\
\

* If the forward contract price is $31 (F_0(1) < 31.50)$, the profitable arbitrage is to buy the forward and sell short an Acme share at $t = 0$. The proceeds of the short sale, $30$, can be invested at the risk-free rate to produce $30x(1.05) = 31.50$ at $t = 1$. The forward contract requires the purchase of a share of Acme for $31$, which the investor can return to close out the short position. The profit to an arbitrageur is $31.50 – 31 = 0.50$. With no cash investment at $t = 0$, the investor receives an arbitrage profit of $0.50$ at $t = 1$.

When the forward price is "too high," the arbitrage is to sell the forward and buy the underlying asset. When the forward price is "too low," the arbitrage is to buy the forward and sell (short) the underlying asset. In either case, the actions of arbitrageurs will move the forward price toward its no-arbitrage level until arbitrage profits are no longer possible.


**Replication** refers to creating a portfolio with cash market transactions that has the same payoffs as a derivative for all possible future values of the underlying.

Our arbitrage example for Acme forwards will serve to illustrate replication.

A long forward on an Acme share can be replicated by borrowing 30 at 5% to purchase an Acme share, and repaying the loan on the settlement date of the forward. At settlement $(t = 1)$, the payoff on the replication is $S_1 – 30(1.05) = S_1 – 31.50$ (value of one share minus the repayment of the loan), the same as the payoff on a long forward at 31.50, for any value of Acme shares at settlement.

A short forward on an Acme share can be replicated by shorting an Acme share and investing the proceeds of $30$ at 5\%. At settlement the investor receives $31.50$ from the investment of short sale proceeds, and must buy a share of Acme for $S_1$. The payoff on the replicating portfolio is $31.50 – S_1$, the same as the payoff on a short forward at $31.50$, for any value of Acme shares at settlement.

These replications allow us to calculate the no-arbitrage forward price of an asset, just as we did in our example using Acme shares. Because our replicating portfolio for a long forward has the same payoff as a long forward at $\text{time = T}$, the payoff at settlement on a portfolio that is long the replicating portfolio and short the forward must be zero to prevent arbitrage. For this strategy, when the forward is priced at its no-arbitrage value the payoff at $\text{time = T}$ is:

$S_T – S_0(1 + R_f)T – [S_T – F_0(T)] = 0$

so that 

$S_0(1 + R_f)T + F_0(T) = 0$ 

and 

$F_0(T) = S_0(1 + R_f)T$.

For a portfolio that is short the replicating portfolio and long the forward, the payoff at time $T$ is:

$S_0(1 + R_f)T– S_T + [S_T – F_0(T)] = 0$

so that 

$S_0(1 + R_f)T – F_0(T) = 0$ 

and 

$F_0(T) = S_0(1 + R_f)T$.

The forward price that will prevent arbitrage is $S_0(1 + R_f)T$, just as we found in our example of a forward contract on an Acme share.

### LOS 71.b: Explain the difference between the spot and expected future price of an underlying and the cost of carry associated with holding the underlying asset.

When we derived the no-arbitrage forward price for an asset as $F_0(T) = S_0(1 + Rf)T$, we assumed there were no benefits of holding the asset and no costs of holding the asset, other than the opportunity cost of the funds to purchase the asset (the risk-free rate of interest).

Any additional costs or benefits of holding the underlying asset must be accounted for in calculating the no-arbitrage forward price. There may be additional costs of owning an asset, especially with commodities, such as storage and insurance costs. For financial assets, these costs are very low and not significant.

There may also be monetary benefits to holding an asset, such as dividend payments for equities and interest payments for debt instruments. Holding commodities may have non-monetary benefits, referred to as **convenience yield**. If an asset is difficult to sell short in the market, owning it may convey benefits in circumstances where selling the asset is advantageous. For example, a shortage of the asset may drive prices up temporarily, making sale of the asset in the short term profitable.

We denote the present value of any costs of holding the asset from time 0 to settlement at time $T$ (e.g., storage, insurance, spoilage) as $PV_0$(cost), and the present value of any cash flows from the asset or convenience yield over the holding period as $PV_0$(benefit).

Consider first a case where there are storage costs of holding the asset, but no benefits. For an asset with no costs or benefits of holding the asset, we established the no-arbitrage forward price as $S_0(1 + R_f)T$, the cost of buying and holding the underlying asset until time $T$. When there are storage costs to hold the asset until time $T$, an arbitrageur must both buy the asset and pay the present value of storage costs at $t = 0$. This increases the no-arbitrage price of a 1-year forward to $[S_0+ PV_0(\text{cost})](1 + R_f)T$. Here we see that costs of holding an asset increase its no-arbitrage forward price.


Next consider a case where holding the asset has benefits, but no costs. Returning to our example of a 1-year forward on a share of Acme stock trading at 30, now consider the costs of buying and holding an Acme share that pays a dividend of $\$1$ during the life of the forward contract. In this case, an arbitrageur can now borrow the present value of the dividend (discounted at $R_f$), and repay that loan when the dividend is received. The cost to buy and hold Acme stock with an annual dividend of $\$1$  is $[30 – PV_0(1)](1.05) = 30(1.05) – 1$. 

*This illustrates that benefits of holding an asset decrease its no-arbitrage forward price.*





The no-arbitrage price of a forward on an asset that has both costs and benefits of holding the asset is simply 

$$[S_0 + PV_0 \text{(costs)} – PV_0\text{(benefit)}](1 + R_f)T$$

We can also describe these relationships when costs and benefits are expressed as continuously compounded rates of return. Recall from Quantitative Methods that given a stated annual rate of r with continuous compounding, the effective annual return is $e^r – 1$, and the relationships between present and future values of S for a 1-year period are 
\
$FV = Se^{r}$ and $PV = Se^{–r}$. 

For a period of $T$ years, $FV = Se^{rT}$ and $PV = Se^{–rT}$.\
With continuous compounding the following relationships hold:

* With no costs or benefits of holding the underlying asset, the no-arbitrage price of a forward that settles at time $T$ is $S_0e^{rT}$, where $r$ is the stated annual risk-free rate with continuous compounding.

* With storage costs at a continuously compounded annual rate of $c$, the no-arbitrage forward price until time $T$ is $S_0e^{(r+c)T}$.

* With benefits, such as a dividend yield, expressed at a continuously compounded annual rate of $b$, the no-arbitrage forward price is until time $T$ is $S_0e^{(r+c-b)T}$.

<hr>

#### Example: No-arbitrage price with continuous compounding

Consider a stock index trading at 1,550 with a dividend yield of 1.3% (continuously compounded rate) when the risk-free rate is 3% (continuously compounded rate). Calculate the no-arbitrage 6-month forward price of the stock index.

In [4]:
from math import e

no_arb_6m_forward = 1550 * e**((0.03-0.013)*(0.5))
print(round(no_arb_6m_forward,2))

1563.23


#### Answer 
The no-arbitrage price of a long 6-month forward is \
$1,550 × e^{(0.03 – 0.013)(0.5)} = 1,563.23.$



#### Forward Contracts on Currencies

Recall from Economics that we defined the no-arbitrage price of a forward on a currency as the forward price that satisfies the equality:

$$
\text{forward exchange rate(p/b)} = \frac{1 + \text{interest rate}_{\text{price currency}}}{1+ \text{interest rate}_{\text{base currency}}} × \text{spot exchange rate}
$$

We can use this no-arbitrage forward rate to examine how an arbitrage profit can be made when the exchange rate in a forward contract is greater or less than the no-arbitrage forward exchange rate. The forward exchange rate depends on the spot exchange rate and the *difference* between the interest rates on the base and price currencies.

Consider a situation at $t = 0$ where the risk-free rate in euros is 3%, the risk-free rate in U.S. dollars is 2%, and the current USD/EUR exchange rate is 1.10. We will examine the arbitrage transactions that establish this relationship by looking at the trades for an investor based in the United States that seeks to profit from the higher interest rate on euros. The investor borrows 100 USD for one year at 2%, exchanges the USD for euros, invests the euros for one year at 3%, and then exchanges the resulting euros for USD. At the end of one year the arbitrageur will have 100/1.10 × 1.03 = 93.64 euros and owe 100(1.02) = 102 USD.

If the arbitrageur has a forward contract to buy USD with a price of 1/1.0893 = 0.9180 euros, he can exchange the 93.64 euros for 93.64/0.9180 = 102 USD, which is the amount owed on the original loan of 100 USD. The depreciation of the euro in the forward price just offsets the higher euro interest, and the arbitrage transaction returns zero. With a forward exchange rate greater than 1.0893, the arbitrage would have a profit, and with a forward exchange rate less than 1.0893, an arbitrageur could profit from the opposite transactions.

If we convert the effective annual rates to equivalent stated annual rates with continuous compounding, we get 

$R_{USD} = \ln 1.02 = 1.98\%$ 

and 

$R_{EUR} = \ln 1.03 = 2.96\%$. 

In this case we can say: \
\
$\text{forward exchange rate} = 1.10 \times e^{(0.0198–0.0296)} = 1.0893.$
\
\
or from ECON, the no-arbitrage forward exchange rate:
\
\
$\text{Forward(A/B)} = \Biggr[\frac{1+Rf_A}{1+Rf_B}\Biggr] x \text{Spot (A/B)}$

In [29]:
forwardAB = (1.0198/1.0296) * 1.10

f01 = 1.10 * e**(0.0198-0.0296)
print(round(f01,4))

1.0893


<hr>

## Reading 71: Key Concepts

<hr>

### LOS 71.a

Valuation of derivative securities is based on a no-arbitrage condition. When the forward price is too high, the arbitrage is to sell the forward and buy the underlying asset. When the forward price is too low, the arbitrage is to buy the forward and sell short the underlying asset. Arbitrage will move the forward price toward its no-arbitrage level.

Replication refers to creating a portfolio with cash market transactions that has the same payoffs as a derivative for all possible future values of the underlying. Replication allows us to calculate the no-arbitrage forward price of an asset.

### LOS 71.b

Assuming no costs or benefits of holding the underlying asset, the forward price that will prevent arbitrage is the spot price compounded at the risk-free rate over the time until expiration.

The cost of carry is the benefits of holding the asset minus the costs of holding the asset.

Greater costs of holding an asset increase its no-arbitrage forward price.

Greater benefits of holding an asset decrease its no-arbitrage forward price.

<hr>

<IMG SRC="https://github.com/PachaTech/CFA-Level-1/blob/main/Derivatives/Module%2071/pics/1.png?raw=true">

<IMG SRC="https://github.com/PachaTech/CFA-Level-1/blob/main/Derivatives/Module%2071/pics/2.png?raw=true">

<IMG SRC="https://github.com/PachaTech/CFA-Level-1/blob/main/Derivatives/Module%2071/pics/3.png?raw=true">


<hr>


## Quiz


<hr>

**Question 1**\
Derivatives pricing models use the risk-free rate to discount future cash flows because these models:


**Answer**\
are based on portfolios with certain payoffs.

**Explanation**\
Derivatives pricing models use the risk-free rate to discount future cash flows (risk-neutral pricing) because they are based on constructing arbitrage relationships that are theoretically riskless.
\
(Module 71.1, LOS 71.a)

<hr>

**Question 2**\
Arbitrage prevents:

**Answer**\
two assets with identical payoffs from selling at different prices.

**Explanation**\
Arbitrage forces two assets with the same expected future value to sell for the same current price. 
\
(Module 71.1, LOS 71.a)


<hr>

**Question 3**\
The underlying asset of a derivative is *most likely* to have a convenience yield when the asset:


**Answer**\
is difficult to sell short.

**Explanation**\
Convenience yield refers to nonmonetary benefits from holding an asset. One example of convenience yield is the advantage of owning an asset that is difficult to sell short when it is perceived to be overvalued. Interest and dividends are monetary benefits. Storage and insurance are carrying costs. 
\
(Module 71.1, LOS 71.b)

<hr>

**Question 4**\
An investor can replicate a forward on a stock that pays no dividends by:

**Answer**\
borrowing at the risk-free rate to buy the underlying.

**Explanation**\
Borrowing $S_0$ at $R_f$ to buy the underlying asset at $S_0$ has a zero cost and pays the spot price of the underlying asset minus the loan repayment of at $time = T$ of $S_0(1 + R_f)^T$, which is the same payoff as a long forward at $F_0 = S_0(1 + R_f)^T$, the no-arbitrage forward price. 
\
(Module 71.1, LOS 71.a)

<hr>

**Question 5**\
The forward price of a commodity will *most likely* be equal to the current spot price if the:


**Answer**\
convenience yield is equal to the risk-free rate plus storage costs as a percentage.

**Explanation**\
When the opportunity cost of funds $(R_f)$ and storage costs just offset the benefits of holding the commodity, the no-arbitrage forward price is equal to the current spot price of the underlying commodity. 
\
(Module 71.1, LOS 71.b)